adminSiteClient/EditorMarimekkoTab.tsx

```diff
@@ -0,0 +1,116 @@
+        const { grapher } = this.props
+        if (grapher.excludedEntities === undefined) {
+            grapher.excludedEntities = []
+        }
+
+        const entityId = grapher.table.entityNameToIdMap.get(entity)!
+        if (grapher.excludedEntities.indexOf(entityId) === -1)
+            grapher.excludedEntities.push(entityId)
+    }
+
+    @action.bound onXOverrideYear(value: number | undefined) {
+        this.props.grapher.xOverrideTime = value
+    }
+
+    @action.bound onUnexcludeEntity(entity: string) {
+        const { grapher } = this.props
+        if (!grapher.excludedEntities) return
+
+        const entityId = grapher.table.entityNameToIdMap.get(entity)
+        grapher.excludedEntities = grapher.excludedEntities.filter(
+            (e) => e !== entityId
+        )
+    }
+
+    render() {
+        const { excludedEntityChoices } = this
+        const { grapher } = this.props
+
+        return (
+            <div className="EditorScatterTab">
+                <Section name="Filtering">
+                    <NumberField
+                        label="Override X axis target year"
+                        value={grapher.xOverrideTime}
+                        onValue={debounce(this.onXOverrideYear, 300)}
+                        allowNegative
+                    />
+
+                    <Toggle
+                        label="Exclude entities that do not belong in any color group"
+                        value={!!grapher.matchingEntitiesOnly}
+                        onValue={action(
+                            (value: boolean) =>
+                                (grapher.matchingEntitiesOnly =
+                                    value || undefined)
+                        )}
+                    />
+                    <SelectField
+                        label="Exclude individual entities"
+                        placeholder="Select an entity to exclude"
+                        value={undefined}
+                        onValue={(v) => v && this.onExcludeEntity(v)}
+                        options={excludedEntityChoices}
+                    />
+                    {this.excludedEntityNames && (
+                        <ul className="excludedEntities">
+                            {this.excludedEntityNames.map((entity) => (
+                                <li key={entity}>
+                                    <div
+                                        className="clickable"
+                                        onClick={() =>
+                                            this.onUnexcludeEntity(entity)
+                                        }
+                                    >
+                                        <FontAwesomeIcon icon={faMinus} />
+                                    </div>
+                                    {entity}
+                                </li>
+                            ))}
+                        </ul>
+                    )}
```

Comment:
As I suggested below, let's make this available for all charts? Filtering the table would happen on the Grapher instead of the individual chart classes, and we can move this to the Customize tab so that it can be used for any chart.

(This is just a suggestion that can be postponed for a later PR in case you find it better to deal with Marimekko issues only here, that might be better.)